tests/test_pytypes.py

```diff
@@ -190,7 +190,8 @@ def test_constructors():
     """C++ default and converting constructors are equivalent to type calls in Python"""
     types = [str, bool, int, float, tuple, list, dict, set]
     expected = {t.__name__: t() for t in types}
-    assert m.default_constructors() == expected
+    default_constructed = m.default_constructors()
+    assert default_constructed == expected
```

Comment:
What's the reason for this change?

Reply:
Reused further below for the new asserts.

Reply:
Should've spotted that.